cla_frontend/settings/base.py

```diff
@@ -193,8 +193,23 @@ def sentry_traces_sampler(sampling_context):
     "cla_auth.middleware.ZoneMiddleware",
     "core.middleware.Cla401Middleware",
     "csp.middleware.CSPMiddleware",
+    "django_cookies_samesite.middleware.CookiesSameSite",
+    "djangosecure.middleware.SecurityMiddleware",
 )
 
+DISABLE_SAMESITE_MIDDLEWARE = os.environ.get("DISABLE_SAMESITE_MIDDLEWARE", "False") == "True"
+
+if not DISABLE_SAMESITE_MIDDLEWARE:
+    MIDDLEWARE_CLASSES = MIDDLEWARE_CLASSES + ("django_cookies_samesite.middleware.CookiesSameSite",)
+else:
+    MIDDLEWARE_CLASSES = MIDDLEWARE_CLASSES
```

Comment:
I don't think you need this else as you are assigning a variable back onto itself